docker-compose.override.yml

```
@@ -0,0 +1,7 @@
+version: '3.7'
+services:
+  gatsby:
+    volumes:
+      - .:/usr/src/app
+    ports:
+      - '${GATSBY_PORT}:8000'
```

Could you add a default value 
```suggestion
      ${GATSBY_PORT:-8000}
```